src/views/activity/ActivityBrowser.vue

```diff
@@ -1,56 +0,0 @@
-<template lang="pug">
-div.mt-3
-  // TODO: Add back option to select a specific browser bucket
-  div(v-if="browserBuckets.length <= 0")
-    h6 No browser buckets available
-    small
-      | This feature requires a browser watcher.
-      | You can find a list of all watchers in #[a(href="https://activitywatch.readthedocs.io/en/latest/watchers.html") the documentation].
```

Comment:
It's a shame that this kind of notice is not there anymore.

Reply:
Yeah, I thought a bit about this since the same thing applies for the editor view.

Basically it would be nice if we'd have a way to indicate *why* a visualization cannot be selected, or alternatively, always let users select visualizations but then show a notice per-visualization if the requisite data is missing.

Reply:
Done!